meta-mender-imx6ul/templates/bblayers.conf.append

```diff
@@ -0,0 +1,11 @@
+
+# i.MX Yocto Project Release layers
+BBLAYERS += "${BSPDIR}/sources/meta-fsl-bsp-release/imx/meta-bsp"
+BBLAYERS += "${BSPDIR}/sources/meta-fsl-bsp-release/imx/meta-sdk"
+
+BBLAYERS += "${BSPDIR}/sources/meta-openembedded/meta-networking"
+BBLAYERS += "${BSPDIR}/sources/meta-openembedded/meta-python"
+
+# Mender layer
+BBLAYERS += "${BSPDIR}/sources/meta-mender/meta-mender-core"
+BBLAYERS += "${BSPDIR}/sources/meta-mender-community/meta-mender-imx6ul"
```

Comment:
This can probably skipped if you do something similar to the [fsl-setup-mender.sh](https://github.com/mendersoftware/meta-mender-community/blob/5adaa8c5cbe332600171589476038ea014fb9268/meta-mender-imx/scripts/fsl-setup-mender.sh) script.

Also we generally also include `meta-mender-demo` in these reference builds.